### DISCUSSION TITLE
是不是新版本的网易云没法用了啊

### DISCUSSION THREAD

**Discussion by chizixiansheng** [1715173329/UnblockNeteaseMusic#877] [Q&A]

**Comment by 1715173329** [comment]
请咨询算命先生以解惑。

**Comment by chizixiansheng** [comment]
么么哒，收到了。

**Comment by chizixiansheng** [comment]
> 请咨询算命先生以解惑。

emmm 骚瑞 可能是我提问的方式不太对？

**Comment by pan93412** [comment]
> > 请咨询算命先生以解惑。
> 
> emmm 骚瑞 可能是我提问的方式不太对？

建議試試看目前 repo 裡面最新的版本，有嘗試修正一些與此相關的問題。

**Reply by 1715173329** [reply]
btw 鉴于最近的兼容性更新，我们是否应当为此建立一个新的 rc 版本？

**Comment by whosydd** [comment]
你可以尝试使用2.9.10版本，或许有奇效

**Comment by ssssshql** [comment]
用旧版

**Comment by pikachullk** [comment]
或者直接用微软商店里面的UWP版嘛~是旧版的界面~但是现在能用~

**Comment by 1715173329** [comment]
本项目已经支持最新版本，但必须信任证书。
由于缺乏任何有效信息，此 issue 将被转为 discussion。

**Reply by pikachullk** [reply]
天灵大佬，我试了试最新版的PC客户端~代理里面指定了docker的latest容器的IP和端口~
运行参数是：-o ytdlp kuwo kugou -p 30966 -s -e http://music.163.com
信任了证书~用UWP版加命令是可以的~exe版运行后显示没有网络
安卓用急速版也是OK的

**Reply by 1715173329** [reply]
新版客户端必须使用 HTTPS。

**Reply by haojunsui** [reply]
更新之后网页使用https地址下载pac正常，但是把端口输入网易云音乐之后依旧显示无网络连接

网易云音乐 ver 2.10.5

**Reply by pikachullk** [reply]
> 新版客户端必须使用 HTTPS。

我以为是客户端的代理必须https~原来是entrypoint用https地址~可以了~感谢天灵大佬~

**Reply by 1715173329** [reply]
> 更新之后网页使用https地址下载pac正常，但是把端口输入网易云音乐之后依旧显示无网络连接
> 
> 网易云音乐 ver 2.10.5

https://github.com/UnblockNeteaseMusic/server/issues/695#issuecomment-1257192357
据说是网易云音乐自身问题，你可以卸载重装试试。
